lib/i18n/english/tags/blog-composer.js

```diff
@@ -0,0 +1,23 @@
+export default {
+      "titlePlaceholder":"Enter your post title here...",
+      "saveLinkText" : "Save",
+      "savedLinkText" : "Saved",
+      "publishLinkText" : "Publish",
+      "unpublishLinkText" : "Unpublish",
+      "loginButtonText" : "Login to start a post",
+      "dummyUserText" : "Unnamed",
+      "dummyContentText" : "Dummy content...",
+      "linkPromptText":"Enter the link URL",
+      "imagePromptText":"Enter the image URL",
+      "editProgessText" : "Saving...",
+      "editErrorText":"Couldn't be saved!",
+      "publishProgressText" : "Publishing...",
+      "publishErrorText" : "Couldn't be published!",
+      "unpublishProgressText" : "unPublishing...",
```

Comment:
"unpublishProgressText" : "unPublishing..."

This is wrong. Why are you using camelCase for the text that will be read by end user. "unPublishing..." Should be "Unpublishing..."